apps/modernization-ui/src/options/operator/operators.ts

```diff
@@ -0,0 +1,26 @@
+import { Selectable, asSelectable, findByValue } from 'options';
+
+const STARTS_WITH_OPERATOR = asSelectable('S', 'Starts with');
```

Comment:
@stevegsa please let me know what the actual values will be for these operators so I can fix it here.